src/components/Beers.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import { Route, Link, Switch } from 'react-router-dom'
+
+export default class Beers extends Component {
+
+    state = {
+        beers: ''
+    }
+
+    componentDidMount() {
+        fetch('https://ih-beers-api2.herokuapp.com/beers')
+          .then(res => res.json())
+          .then(result => {
+            console.log(result)
+            this.setState({beers: result})
+          })
+          .catch(e => console.log(e))
+    }
+
+    render() {
+        return (
+            <div>
+                <Header />
+
+                {this.state.beers === '' ? 'Beers are loading' : 
+                this.state.beers.map((beer, index) => (
+                    <div key={index}>
+                    <Link className="link-none" to={"/" + beer._id}>
```

Comment:
You can also use **template literals** like this `<Link className="link-none" to={`/${beer._id}`}>`